includes/class-wc-payments-utils.php

```diff
@@ -887,4 +887,47 @@ public static function get_wpcore_request_class(): string {
 			? '\\WpOrg\\Requests\\Requests'
 			: '\\Requests';
 	}
+
+	/**
+	 * Returns a merchant-friendly description of the dispute reason.
+	 *
+	 * This mapping is duplicated in client/disputes/strings.ts and on Server.
+	 *
+	 * @param string $reason The dispute reason.
+	 *
+	 * @return string
+	 */
+	public static function get_dispute_reason_description( string $reason ): string {
+		switch ( $reason ) {
+			case 'bank_cannot_process':
+				return __( 'Bank can not process', 'woocommerce-payments' );
```

Comment:
I think this should be `Bank cannot process` which is consistent with `client/disputes/strings.ts` and the changes you've proposed in https://github.com/Automattic/woocommerce-payments-server/pull/3436.

https://github.com/Automattic/woocommerce-payments/blob/7cef6df74173a4d6e278f08ec4346b588e48bb5d/client/disputes/strings.ts#L22

```suggestion
				return __( 'Bank cannot process', 'woocommerce-payments' );
```